packages/avatar/src/vaadin-avatar.js

```diff
@@ -4,8 +4,8 @@
  * This program is available under Apache License Version 2.0, available at https://vaadin.com/license/
  */
 import { PolymerElement, html } from '@polymer/polymer/polymer-element.js';
+import { ElementMixin } from '@vaadin/component-base';
```

Comment:
Let's use direct imports as we do with other mixins:

```suggestion
import { ElementMixin } from '@vaadin/component-base/src/element-mixin.js';
```

Please update `.d.ts` file accordingly.